### PR TITLE
docs: update Nextjs docs

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update Nextjs docs |
| 🇨🇳 Chinese | docs: update Nextjs docs |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at b6c576a</samp>

Improved the documentation for using `antd` with `Next.js` in both English and Chinese. Added tips and notes to help developers resolve style issues and improve performance.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at b6c576a</samp>

*  Moved comment to a new line for readability and consistency ([link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-f40a07db5e50bc714a69751497cc62a602271f477eabf7c895f2fa9950461ae4L36-R38), [link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-b532268d5c5f44bba9caadf328fe6567e31ba5fc7c36cfe6eec648bcf1b36489L36-R38))
*  Added note to warn about potential issue of installing `@ant-design/cssinjs` with a different version than `antd` ([link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-f40a07db5e50bc714a69751497cc62a602271f477eabf7c895f2fa9950461ae4R61-R64), [link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-f40a07db5e50bc714a69751497cc62a602271f477eabf7c895f2fa9950461ae4R167-R170), [link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-b532268d5c5f44bba9caadf328fe6567e31ba5fc7c36cfe6eec648bcf1b36489R61-R64), [link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-b532268d5c5f44bba9caadf328fe6567e31ba5fc7c36cfe6eec648bcf1b36489R167-R170))
*  Translated note into Chinese for `docs/react/use-with-next.zh-CN.md` ([link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-b532268d5c5f44bba9caadf328fe6567e31ba5fc7c36cfe6eec648bcf1b36489R61-R64), [link](https://github.com/ant-design/ant-design/pull/45907/files?diff=unified&w=0#diff-b532268d5c5f44bba9caadf328fe6567e31ba5fc7c36cfe6eec648bcf1b36489R167-R170))

### CODE REVIEW

**Review by afc163 — APPROVED** [review]

File: docs/react/use-with-next.en-US.md [hunk 1]
```diff
@@ -159,6 +163,10 @@ If you are using the Pages Router in Next.js and using antd as your component li
 
 1. Install `@ant-design/cssinjs`
 
+> Notes for developers
+>
+> Please note that when you install `@ant-design/cssinjs`, you must ensure that the version is consistent with the version of `@ant-design/cssinjs` in `node_modules` of `antd`, otherwise, multiple React instances will appear, resulting in ctx being unable to be read correctly.
```
Comment (afc163) [inline docs/react/use-with-next.en-US.md:168]: 如何确定版本一致，给个方法。

Comment (li-jia-nan) [reply docs/react/use-with-next.en-US.md:168]: 好像没有什么很好的方法，只能让用户手动检查，看了一下，不管是 `yarn` 的 `resolutions`，还是 `pnpm` 的 `overrides`，都是只能写死版本号，没办法动态指定版本

Comment (afc163) [reply docs/react/use-with-next.en-US.md:168]: > 只能让用户手动检查

用户如何手动检查，不是所有用户都知道怎么操作的。

Comment (li-jia-nan) [reply docs/react/use-with-next.en-US.md]: update. 加了个 Tips

File: docs/react/use-with-next.zh-CN.md [hunk 2]
```diff
@@ -59,7 +60,7 @@ export default Home;
 
 > 开发者注意事项：
 >
-> 请注意，安装 `@ant-design/cssinjs` 时必须确保版本号跟 `antd` 的 `node_modules` 中的 `@ant-design/cssinjs` 版本保持一致，否则会出现多个 React 实例，导致无法正确的读取 ctx。
+> 请注意，安装 `@ant-design/cssinjs` 时必须确保版本号跟 `antd` 本地的 `node_modules` 中的 `@ant-design/cssinjs` 版本保持一致，否则会出现多个 React 实例，导致无法正确的读取 ctx。（Tips: 你可以通过 `npm ls @ant-design/cssinjs` 命令查看本地版本）
```
Comment (afc163) [inline docs/react/use-with-next.zh-CN.md:62]: 截个实际场景的图试试。

Comment (li-jia-nan) [reply docs/react/use-with-next.zh-CN.md:62]: image

是说把这个截图直接放进文档吗？

Comment (afc163) [reply docs/react/use-with-next.zh-CN.md:62]: 找一个 next.js 项目，装一个 antd，再装一个 @ant-design/cssinjs，然后 npm ls 一下，截图，把两个 @ant-design/cssinjs 版本框出来。

Comment (li-jia-nan) [reply docs/react/use-with-next.zh-CN.md:62]: - #45929

**Pull request merged**